src/DependencyInjection/SchedulerBundleExtension.php

```diff
@@ -968,4 +969,9 @@ private function registerDataCollector(ContainerBuilder $container): void
             ])
         ;
     }
+
+    public function getConfiguration(array $config, ContainerBuilder $container): ConfigurationInterface
```

Comment:
Consider adding `{@inheritdoc}`

Reply:
I will replace :configurationInterface by the PHPdoc you are right !

Reply:
That's not the idea, the return typehint can stay but you need to add the phpdoc for `{@inheritdoc}` 🙂